etc/get_bson_version.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+with open('Package.resolved', 'r') as f:
```

Comment:
JSON parsing is where I draw the line for what I will try to do in a bash script lol